### DISCUSSION TITLE
How to ensure the pieces as a Block that not disconnected？

### DISCUSSION THREAD

**Discussion by ovsexia** [flbulgarelli/headbreaker#48] [Q&A] [answered]
Hi!
I put multiple pieces together that I get a Block named A.
When I want to move the Block A, pieces are often disconnected from other pieces.

Would it be possible to add a feature like this:
When I hold  Ctrl and drag the Block A, is guaranteed not to disconnect the pieces?

Thank you.

**Comment by flbulgarelli** [comment, marked as answer]
Hi!

Currently the only way to keep pieces connected during drag is gesture-based - you need to push the group in a direction that is not free of pieces.

However, I have been experimenting with extending gestures with keyboard strokes, in a similar way to your proposal.  Perhaps you can try this branch:  https://github.com/flbulgarelli/headbreaker/pull/47.

**Reply by ovsexia** [reply]
Thank. It works.

**Comment by flbulgarelli** [comment]
I have released a new version of `headbreaker` that ships with that functionality. You can check it here: https://flbulgarelli.github.io/headbreaker/#keyboard-gestures
